File: ios/MobilePassSDK/Model/QRCodeMatch.swift
//
//  QRCodeMatch.swift
//  MobilePassSDK
//
//  Created by Erinc Cakir on 18.08.2021.
//

import Foundation

struct QRCodeMatch: Codable {
    var accessPointId:  String
    var qrCode:         ResponseAccessPointListQRCode
}

File: android/mobilepasssdk/src/main/java/com/armongate/mobilepasssdk/model/QRCodeMatch.java
package com.armongate.mobilepasssdk.model;

import com.armongate.mobilepasssdk.model.response.ResponseAccessPointListQRCode;

public class QRCodeMatch {
    public String                           accessPointId;
    public ResponseAccessPointListQRCode    qrCode;

    public QRCodeMatch(String accessPointId, ResponseAccessPointListQRCode qrCode) {
        this.accessPointId = accessPointId;
        this.qrCode = qrCode;
    }
}
